kafka/producer_test.go

```diff
@@ -176,6 +176,52 @@ func TestProducerAPIs(t *testing.T) {
 	if offsets != nil {
 		t.Errorf("OffsetsForTimes() failed but returned non-nil Offsets: %s\n", offsets)
 	}
+
+	// Purge messages that are not delivered to kafka
+	unreachableProducer, err := NewProducer(&ConfigMap{
+		"bootstrap.servers": "unreachable:9092",
+		"message.timeout.ms": 50,
```

Comment:
This timeout is pretty low, it is not certain that you'll be able to call Purge() prior to this timeout hitting, depending on system load (such as on CI).

Why not use the default timeout since all messages are purged anyway?